Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -1776,9 +1777,30 @@ def prepare_NANO(self, stepSpec = '' ):
         print(f"in prepare_nano {stepSpec}")
         ''' Enrich the schedule with NANO '''
         _,_nanoSeq,_nanoCff = self.loadDefaultOrSpecifiedCFF(stepSpec,self.NANODefaultCFF,self.NANODefaultSeq)
-        self.scheduleSequence(_nanoSeq,'nanoAOD_step')
-        custom = "nanoAOD_customizeCommon"
-        self._options.customisation_file.insert(0,'.'.join([_nanoCff,custom]))
```

Comment:
NB : inserting at 0, versus appending has undesirable effects. #44908 helps once all is in autoNano